documentation/modules/ROOT/pages/ports.adoc

```diff
@@ -54,7 +54,7 @@ You now got this output:
 [.console-output]
 [source,text]
 ----
-hello
+Hello RESTEasy
```

Comment:
Hi, it looks like now the default is "Hello from RESTEasy Reactive".  Could you update it?